src/server_manager/model/server.ts

```diff
@@ -122,8 +122,6 @@ export interface ManagedServerHost {
   getRegionId(): RegionId;
   // Deletes the server - cannot be undone.
   delete(): Promise<void>;
-  // Returns the virtual host ID.
-  getHostId(): string;
 }
```

Comment:
Can we update the Server#getId method to mention that this ID is unique across clouds?

Reply:
Done